### PR TITLE
Update Readme with latest Project details

### PR DESCRIPTION
**Pull request by aruneshchandra**
Updating Readme with latest project details

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,45 @@
...
+
+**Branches**
 
-~
+This repoistory contains node sources from Node versions 0.10, 0.12, 6.2 and
+Node-ChakraCore version 7.0, whith addition of ABI stable Node APIs. The branches
+are named according to the node versions that have been enabled with NAPI support. 
 
+**NAPI enabled modules**
+* leveldown
```
Comment (sampsongao) [inline README.md]: Maybe we want to include a link to converted modules code.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,45 @@
...
+* nanomsg
+
+**How to get involved**
+* Convert a native module to use ABI stable APIs and report issues on conversion and performance;
+* Port ABI stable APIs to your fork of Node and let us know if there are gaps;
+* Review the [roadmap](https://github.com/nodejs/abi-stable-node/issues/18) and see how can you
+can help accelerate this project.
+
+Hangout link for weekly standup
+https://plus.google.com/u/0/events/c0eevtrlajniu7h8cjrdk0f56c8?authkey=COH04YCalJS8Ug
```
Comment (sampsongao) [inline README.md:46]: To participate in this event, they need to request an invite. We should mention that or I can change the setting of the event to allow people without invitation to take part in.

File: README.md (hunk 1 above)
Comment (aruneshchandra) [reply README.md]: Good point - I don't know where they are. Can you point me to it ? Or better yet just add em to the PR.

File: README.md [hunk 3]
```diff
@@ -1,18 +1,45 @@
-ABI Stable Node.js Proof of Concept
-===================================
+# Node.js API (NAPI)
+This repository is the home for ABI Stable Node API project. The goal of this
+project is to provide stable Node API for native module developers. NAPI aims
```
Comment (mhdawson) [inline README.md]: "stable Node" -> "a stable Node"

File: README.md [hunk 4]
```diff
@@ -1,18 +1,45 @@
-ABI Stable Node.js Proof of Concept
-===================================
+# Node.js API (NAPI)
+This repository is the home for ABI Stable Node API project. The goal of this
```
Comment (mhdawson) [inline README.md]: add (NAPI) before the period

File: README.md [hunk 5]
```diff
@@ -1,18 +1,45 @@
...
 
-~
+This repoistory contains node sources from Node versions 0.10, 0.12, 6.2 and
+Node-ChakraCore version 7.0, whith addition of ABI stable Node APIs. The branches
+are named according to the node versions that have been enabled with NAPI support. 
 
+**NAPI enabled modules**
+* leveldown 
+* nanomsg
+
```
Comment (mhdawson) [inline README.md:38]: add
[sqlite3](https://github.com/mhdawson/node-sqlite3) (in progress)

File: README.md (hunk 2 above)
Comment (mhdawson) [reply README.md:46]: I'd suggest we just allow people in without an invite.

File: README.md (hunk 5 above)
Comment (mhdawson) [reply README.md:38]: Also add 
** Testing **
In addition to running the tests in the converted tests.  We also have

* Converted version of the NAN examples that we always keep working
  [abi-stable-node-addon-examples](https://github.com/nodejs/abi-stable-node-addon-examples)

* converted version of the core addons tests - ex: 
[https://github.com/nodejs]/abi-stable-node/tree/api-prototype-6.2.0/test/addons-abi(https://github.com/nodejs/abi-stable-node/tree/api-prototype-6.2.0/test/addons-abi) 
  which can be run with ```make test addons-abi```

**Review by mhdawson — COMMENTED** [review]
Left a number of specific comments but in general loooks good.

**Comment by aruneshchandra** [comment]
Updated with review feedbacks.

File: README.md [hunk 6]
```diff
@@ -1,18 +1,57 @@
...
-https://plus.google.com/u/0/events/c0eevtrlajniu7h8cjrdk0f56c8?authkey=COH04YCalJS8Ug
+For more details on design of the API, please check out the 
+[node-eps](https://github.com/nodejs/node-eps/pull/20) covering this effort. 
+
+The current shape of the API can be found in header file 
+[node_jsvmapi.h](https://github.com/nodejs/abi-stable-node/blob/api-prototype-6.2.0/src/node_jsvmapi.h)
+
+**Branches**
+
+This repoistory contains node sources from Node versions 0.10, 0.12, 6.2 and
```
Comment (digitalinfinity) [inline README.md]: Nit: repository spelling

File: README.md [hunk 7]
```diff
@@ -1,18 +1,57 @@
...
+Node-ChakraCore version 7.0, whith addition of ABI stable Node APIs. The branches
```
Comment (digitalinfinity) [inline README.md]: Nit: "whith" spelling?

File: README.md [hunk 8]
```diff
@@ -1,18 +1,57 @@
...
-~
+**Testing**
 
+In addition to running the tests in the converted modules.  We have also 
+
+* Converted version of the NAN examples that we always keep working
+  [abi-stable-node-addon-examples](https://github.com/nodejs/abi-stable-node-addon-examples)
+
+* Converted version of the core addons tests - ex: 
+[https://github.com/nodejs]/abi-stable-node/tree/api-prototype-6.2.0/test/addons-abi(https://github.com/nodejs/abi-stable-node/tree/api-prototype-6.2.0/test/addons-abi) which can be run with ```make test addons-abi```
```
Comment (mhdawson) [inline README.md]: This link is not quite right.  [https://github.com/nodejs]/abi-stable-node/tree/api-prototype-6.2.0/test/addons-abi(https://github.com/nodejs/abi-stable-node/tree/api-prototype-6.2.0/test/addons-abi)

should be  (Believe just missing closing ])

[https://github.com/nodejs]/abi-stable-node/tree/api-prototype-6.2.0/test/addons-abi](https://github.com/nodejs/abi-stable-node/tree/api-prototype-6.2.0/test/addons-abi)

**Review by mhdawson — APPROVED** [review]
LGTM with one comment on broken link.

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**